inscriptions/ISic000481.xml

```diff
@@ -185,15 +190,34 @@
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
-            </div>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AR">
+                <ab>
+                    <lb n="1"/>pro beatitudine
+                    <lb n="2"/>temporum <expan><abbr>dd</abbr><ex>ominorum</ex></expan> <expan><abbr>nn</abbr><ex>ostrorum</ex></expan>
+                    <lb n="3"/><name>Constanti</name> <g ref="#interpunct">·</g> et
+                    <lb n="4"/><name>Constantis</name> <name><expan><abbr>AAuugg</abbr><ex>ustorum</ex></expan></name>
+                    <lb n="5"/>stationem a solo fece
+                    <lb n="6" break="no"/> runt <name>Vitrasius</name> <name>Orfitus</name> et <name><expan><abbr>Fl</abbr><ex>avius</ex></expan></name>
+                    <lb n="7"/><name>Dulcitius</name> <expan><abbr>vv</abbr><ex>iri</ex></expan> <expan><abbr>cc</abbr><ex>larissimi</ex></expan> consulares
+                    <lb n="8"/><expan><abbr>p</abbr><ex>rovinciae</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>S</abbr><ex>iciliae</ex></expan> <g ref="#interpunct">·</g> instante <name><expan>Fl</expan><ex>avio</ex></name> <g ref="#interpunct">·</g> <name>Valeriano</name>
+                    <lb n="9"/><rs type="official" key="ducenarius">ducenario</rs> <rs type="official" key="commissioner">agente</rs> in <expan><abbr>reb</abbr><ex>us</ex></expan> et <rs type="official" key="chief"><expan><abbr>p</abbr><ex>rae</ex><abbr>p</abbr><ex>osito</ex></expan></rs> cursus
+                    <lb n="10"/>publici
+		    </ab> 
+		</div>
             <div type="apparatus">
                 <p><!--external apparatus criticus (if applicable)--></p>
+		    <listApp>
+			    <app>
+				    <note>text of CIL</note>
+			    </app>
+		    </listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="it" resp="#AR">
+                <p>Per la felicità dei tempi dei nostri signori Constante e Costanzo gli Augusti, i viri clarissimi consolari della provincia di Sicilia, Vitrasio Orfito e Flavio Dulcizio, costruirono un posto di guardia dalle fondamenta, sotto il controllo di Flavio Valeriano in qualità di ducenario, commissario imperiale nella pubblica amministraione e comandante del servizio di posta</p>
             </div>
+		<div type="translation" xml:lang="en" resp="#AR">
+			<p>In return for the fortune of the time of our lords Constantis and Constantius, the consuls of the Sicilian Province, Vitrasius Orfitus and Flavius Dulcitius, both men of oustanding merit, constructed a lookout from the ground up. (This was done) under the auspices of Flavius Valerianus, functioning as Ducenarius, commissioner in the public administration and chief of the postal service</p>
```

Comment:
Constantius and Constans?
consulars of the province rather than consuls. 
I think a statio in this context would be something like 'way-station'